src/common/laws/kmpmp_health_system.txt

```diff
@@ -0,0 +1,159 @@
+﻿law_charitable_health_system = {
+	group = lawgroup_health_system
+	icon = "gfx/interface/icons/law_icons/charitable_health_system.dds"
+	
+	progressiveness = 50
+	
+	unlocking_technologies = {
+		medical_degrees
+	}	
+
+	institution = institution_health_system
+	institution_modifier = {
+		state_mortality_mult = -0.03
+
+		interest_group_ig_devout_pol_str_mult = 0.1
+		state_pollution_reduction_health_mult = -0.1
+		state_poor_standard_of_living_add = 0.75
+	}
+	
+	# Don't return to charity if we have something more advanced
+	ai_will_do = {
+		NOR = {
+			has_law = law_type:law_private_health_insurance
+			has_law = law_type:law_public_health_insurance
+		}
+	}
+	
+	pop_support = {
+		value = 0	
+		add = {
+			desc = "POP_POVERTY"			
+			if = {
+				limit = { 
+					owner = { NOT = { has_law = law_type:law_public_health_insurance } }
+				}
+
+				# SoL 14 gives 0.1 pop support, 13 -> 0.2, 12 -> 0.3, ... 5 -> 1.0
+				value = 15
+				subtract = standard_of_living
+				multiply = 0.1
+				min = 0
+				max = 1
+			}					
+		}
+	}
+
+	ai_enact_weight_modifier = { #Petitions
+		value = 0
+		
+		if = {
+			limit = { 
+				has_journal_entry = je_government_petition
+				has_variable = desired_law_var
+                scope:law = var:desired_law_var
+			}
+			add = 750
+		}
+	}
+}
+
+law_private_health_insurance = {
+	group = lawgroup_health_system
+	icon = "gfx/interface/icons/law_icons/private_health_insurance.dds"
+	
+	progressiveness = 75
+	
+	unlocking_technologies = {
+		pharmaceuticals
+	}
+	
+	modifier = {
+		country_institution_health_system_max_investment_add = 2
+	}	
+	
+	institution = institution_health_system
+	institution_modifier = {
+		interest_group_ig_petty_bourgeoisie_pol_str_mult = 0.1
+		state_mortality_wealth_mult = -0.0025
+		state_pollution_reduction_health_mult = -0.1
+		state_rich_standard_of_living_add = 0.6
+		state_middle_standard_of_living_add = 0.3
+	}
+	
+	pop_support = {
+		value = 0	
+		add = {
+			desc = "POP_WEALTH"			
+			# Wealth 20 gives 0.1 pop support, 22 -> 0.2, 24 -> 0.3, ..., 30 -> 0.6
+			value = wealth
+			subtract = 18
+			divide = 2
+			multiply = 0.1
+			min = 0
+			max = 0.6
+		}
+	}
+
+	ai_enact_weight_modifier = { #Petitions
+		value = 0
+		
+		if = {
+			limit = { 
+				has_journal_entry = je_government_petition
+				has_variable = desired_law_var
+                scope:law = var:desired_law_var
+			}
+			add = 750
+		}
+	}
+}
+
+law_public_health_insurance = {
+	group = lawgroup_health_system
+	icon = "gfx/interface/icons/law_icons/public_health_insurance.dds"
+	
+	progressiveness = 100
+	
+	unlocking_technologies = {
+		pharmaceuticals
+	}
+	
+	modifier = {
+		country_institution_health_system_max_investment_add = 2
+	}		
+	
+	institution = institution_health_system
+	institution_modifier = {
+		state_mortality_mult = -0.04
+		state_pollution_reduction_health_mult = -0.15
+		state_standard_of_living_add = 0.3
+		country_institution_health_system_max_investment_add = 1
```

Comment:
das sollte in: law_charitable_health_system

dann aber dort wie hier auch mit modifier und nicht mit institution_modifier